README.md

```diff
@@ -288,6 +288,37 @@ public class FailoverSampleApp2 {
   }
 }
 ```
+
+### Plugin Manager
+
+<div style="text-align:center"><img src="./docs/files/images/plugin_manager_diagram.png" /></div>
+
+Figure shows a simplified workflow of the plugin manager. Starting at the top, when a new query is executed by the application, it is passed into the plugin manager. From the  plugin manager, the query is passed in order of plugins loaded, in this figure, `Custom Plugin A` to `Custom Plugin B` and finally to `Default Plugin` which executes the query and returns the result sets back up the chain.
+Custom plugins can be used to interact with queries, such as adding a feature to monitor a connection's status in the case of `NodeMonitoringConnectionPlugin`. 
+
+By default, `NodeMonitoringConnectionPlugin` is loaded for [Enhanced Failure Monitoring](https://github.com/awslabs/aws-mysql-jdbc#aws-iam-database-authentication). Additional custom plugins can be included and chained together.
```

Comment:
Is this the correct hyperlink?

Reply:
Additional custom plugins can be included and chained together. -> Additional custom plugins can be implemented  and used alongside to existing ones. Plugins can be chained together in a desired order.

Reply:
They're not supposed to work until it has been merged into AWS main

Reply:
But the link is pointing to `aws-iam-database-authentication`

Reply:
Ah, good catch.